src/css/profile/mobile/common/listview.less

```diff
@@ -139,7 +136,8 @@ tau-expandable {
 		.ui-li-text {
 			display: inline-flex;
 			flex-direction: column;
-			padding: 14 * @px_base 4 * @px_base;
+			padding: 14 * @px_base 0;
```

Comment:
Can I omit the trailing 0?

Reply:
I don't understand your suggestion,, why omit the trailing '0'?

Reply:
This is correct for me.